api.planx.uk/modules/analytics/metabase/collection/service.ts

```diff
@@ -0,0 +1,90 @@
+import { MetabaseError, createMetabaseClient } from "../shared/client.js";
+import type { NewCollectionParams } from "./types.js";
+import { toSnakeCase } from "../shared/utils.js";
+
+const client = createMetabaseClient();
+
+export async function authentication(): Promise<boolean> {
+  try {
+    const response = await client.get("/user/current");
+    return response.status === 200;
+  } catch (error) {
+    console.error("Error testing Metabase connection:", error);
+    return false;
+  }
+}
```

Comment:
This also looks unused currently.

I think this should be a middleware added to the route - before proceeding to make additional requests to Metabase, we can first check the API key is valid.